frontend/src/Sidebar.tsx

```diff
@@ -96,7 +100,7 @@ export class Sidebar extends React.Component<PageComponents, PageComponents>{
                         Hello {this.state.loginManager.getLoginStatus().username}!
                     </div>
                 </div>
-                <div className="navbar-page">{selectedComponent?.page}</div>
+                <div className="navbar-page">{selectedComponent?.page}<div className="navbar-footer-container"><Footer/></div></div>
```

Comment:
A little too much nesting in one line, isn't it?
```suggestion
                <div className="navbar-page">
                    {selectedComponent?.page}
                    <div className="navbar-footer-container">
                        <Footer/>
                    </div>
                </div>
```